mkosi/__init__.py

```diff
@@ -168,6 +170,36 @@ class MkosiException(Exception):
     """Leads to sys.exit"""
 
 
+def dictify(f: Callable[..., Generator[Tuple[str, str], None, None]]) -> Callable[..., Dict[str, str]]:
+    def wrapper(*args: Any, **kwargs: Any) -> Dict[str, str]:
+        return dict(f(*args, **kwargs))
+
+    return functools.update_wrapper(wrapper, f)
```

Comment:
I believe the preferred way of doing this is to use `functools.wraps` (https://docs.python.org/3/library/functools.html#functools.wraps)

Reply:
It's equivalent. `functools.wraps` is a decorator and is used when the wrapped function is longer. For a short one like this, `update_wrapper` is fine.